TASK [install node packages] FAILED

I am seeing an error during the provisioning stage for task name 'install node packages'.

> "**stderr**": "npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to \"coffeescript\" (no hyphen)\nnpm

RAW output of failed task below:
```
TASK [install node packages] ***************************************************
fatal: [ofn-devbox]: FAILED! => {"changed": true, "cmd": "bash -lc \"npm install\"", "delta": "0:00:39.322111", "end": "2019-04-27 13:56:49.195491", "msg": "non-zero return code", "rc": 185, "start": "2019-04-27 13:56:09.873380", "stderr": "npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to \"coffeescript\" (no hyphen)\nnpm WARN optional Skipping failed optional dependency /chokidar/fsevents:\nnpm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.8\nnpm ERR! Linux 4.4.0-146-generic\nnpm ERR! argv \"/home/vagrant/.nodenv/versions/5.12.0/bin/node\" \"/home/vagrant/.nodenv/versions/5.12.0/bin/npm\" \"install\"\nnpm ERR! node v5.12.0\nnpm ERR! npm  v3.8.6\nnpm ERR! path ../atob/bin/atob.js\nnpm ERR! code EPROTO\nnpm ERR! errno -71\nnpm ERR! syscall symlink\n\nnpm ERR! EPROTO: protocol error, symlink '../atob/bin/atob.js' -> '/home/vagrant/openfoodnetwork/node_modules/.bin/atob'\nnpm ERR! \nnpm ERR! If you need help, you may report this error at:\nnpm ERR!     <https://github.com/npm/npm/issues>\n\nnpm ERR! Please include the following file with any support request:\nnpm ERR!     /home/vagrant/openfoodnetwork/npm-debug.log", "stderr_lines": ["npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to \"coffeescript\" (no hyphen)", "npm WARN optional Skipping failed optional dependency /chokidar/fsevents:", "npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.2.8", "npm ERR! Linux 4.4.0-146-generic", "npm ERR! argv \"/home/vagrant/.nodenv/versions/5.12.0/bin/node\" \"/home/vagrant/.nodenv/versions/5.12.0/bin/npm\" \"install\"", "npm ERR! node v5.12.0", "npm ERR! npm  v3.8.6", "npm ERR! path ../atob/bin/atob.js", "npm ERR! code EPROTO", "npm ERR! errno -71", "npm ERR! syscall symlink", "", "npm ERR! EPROTO: protocol error, symlink '../atob/bin/atob.js' -> '/home/vagrant/openfoodnetwork/node_modules/.bin/atob'", "npm ERR! ", "npm ERR! If you need help, you may report this error at:", "npm ERR!     <https://github.com/npm/npm/issues>", "", "npm ERR! Please include the following file with any support request:", "npm ERR!     /home/vagrant/openfoodnetwork/npm-debug.log"], "stdout": "openfoodnetwork@1.7.1 /home/vagrant/openfoodnetwork\n├── anymatch@1.3.2  extraneous\n├── async-each@1.0.3  extraneous\n├── base@0.11.2  extraneous\n├── batch@0.5.3  extraneous\n├── bluebird@2.11.0  extraneous\n├── body-parser@1.19.0  extraneous\n├── colors@1.3.3  extraneous\n├── connect@3.6.6  extraneous\n├── core-js@2.6.5  extraneous\n├── decode-uri-component@0.2.0  extraneous\n├── di@0.0.1  extraneous\n├── dom-serialize@2.2.1  extraneous\n├── expand-braces@0.1.2  extraneous\n├── fragment-cache@0.2.1  extraneous\n├── graceful-fs@4.1.15  extraneous\n├── http-proxy@1.17.0  extraneous\n├── is-binary-path@1.0.1  extraneous\n├── is-windows@1.0.2  extraneous\n├── isbinaryfile@3.0.3  extraneous\n├── jasmine-core@2.4.1 \n├── UNMET PEER DEPENDENCY karma@>=0.11.14\n├─┬ karma-chrome-launcher@2.2.0 \n│ ├─┬ fs-access@1.0.1 \n│ │ └── null-check@1.0.0 \n│ └─┬ which@1.3.1 \n│   └── isexe@2.0.0 \n├─┬ karma-coffee-preprocessor@0.3.0 \n│ └── coffee-script@1.12.7 \n├── karma-jasmine@0.3.8 \n├── lodash@3.10.1  extraneous\n├── log4js@0.6.38  extraneous\n├── mime@1.6.0  extraneous\n├── object.pick@1.3.0  extraneous\n├── optimist@0.6.1  extraneous\n├── posix-character-classes@0.1.1  extraneous\n├── readable-stream@2.3.6  extraneous\n├── resolve-url@0.2.1  extraneous\n├── rimraf@2.6.3  extraneous\n├── snapdragon-node@2.1.1  extraneous\n├── socket.io@1.7.4  extraneous\n├── source-map@0.5.7  extraneous\n├── source-map-url@0.4.0  extraneous\n├── to-regex@3.0.2  extraneous\n├── to-regex-range@2.1.1  extraneous\n├── urix@0.1.0  extraneous\n├── use@3.1.1  extraneous\n└── useragent@2.3.0  extraneous", "stdout_lines": ["openfoodnetwork@1.7.1 /home/vagrant/openfoodnetwork", "├── anymatch@1.3.2  extraneous", "├── async-each@1.0.3  extraneous", "├── base@0.11.2  extraneous", "├── batch@0.5.3  extraneous", "├── bluebird@2.11.0  extraneous", "├── body-parser@1.19.0  extraneous", "├── colors@1.3.3  extraneous", "├── connect@3.6.6  extraneous", "├── core-js@2.6.5  extraneous", "├── decode-uri-component@0.2.0  extraneous", "├── di@0.0.1  extraneous", "├── dom-serialize@2.2.1  extraneous", "├── expand-braces@0.1.2  extraneous", "├── fragment-cache@0.2.1  extraneous", "├── graceful-fs@4.1.15  extraneous", "├── http-proxy@1.17.0  extraneous", "├── is-binary-path@1.0.1  extraneous", "├── is-windows@1.0.2  extraneous", "├── isbinaryfile@3.0.3  extraneous", "├── jasmine-core@2.4.1 ", "├── UNMET PEER DEPENDENCY karma@>=0.11.14", "├─┬ karma-chrome-launcher@2.2.0 ", "│ ├─┬ fs-access@1.0.1 ", "│ │ └── null-check@1.0.0 ", "│ └─┬ which@1.3.1 ", "│   └── isexe@2.0.0 ", "├─┬ karma-coffee-preprocessor@0.3.0 ", "│ └── coffee-script@1.12.7 ", "├── karma-jasmine@0.3.8 ", "├── lodash@3.10.1  extraneous", "├── log4js@0.6.38  extraneous", "├── mime@1.6.0  extraneous", "├── object.pick@1.3.0  extraneous", "├── optimist@0.6.1  extraneous", "├── posix-character-classes@0.1.1  extraneous", "├── readable-stream@2.3.6  extraneous", "├── resolve-url@0.2.1  extraneous", "├── rimraf@2.6.3  extraneous", "├── snapdragon-node@2.1.1  extraneous", "├── socket.io@1.7.4  extraneous", "├── source-map@0.5.7  extraneous", "├── source-map-url@0.4.0  extraneous", "├── to-regex@3.0.2  extraneous", "├── to-regex-range@2.1.1  extraneous", "├── urix@0.1.0  extraneous", "├── use@3.1.1  extraneous", "└── useragent@2.3.0  extraneous"]}
	to retry, use: --limit @/vagrant/setup/provision.retry

PLAY RECAP *********************************************************************
ofn-devbox                 : ok=27   changed=8    unreachable=0    failed=1   
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,6 @@
+# Set line endings to LF, even on Windows / OSX.
+# See https://help.github.com/articles/dealing-with-line-endings/
+
+*.sh text eol=lf
+*.rb text eol=lf
+*.yml text eol=lf

--- a/README.md
+++ b/README.md
@@ -3,6 +3,7 @@ This repo is designed to help Windows or Mac users get a dev environment up and 
 #### Setup:
 
 - Install Virtualbox, Vagrant, and Git.
+- Vagrant commands need to be run from an Administrator terminal on Windows.
 - Start the box with the command: `vagrant up` (inside this directory).
 - Wait for the installation to finish...
 - Profit!
@@ -33,6 +34,9 @@ vagrant reload
 
 # Delete the VM:
 vagrant destroy
+
+# Retry the provisioning (setup) stage:
+vagrant up --provision
 ```
 
 You can also connect to the VM via SSH using the command: `vagrant ssh`. 

--- a/setup/provision.yml
+++ b/setup/provision.yml
@@ -107,7 +107,7 @@
         chdir: /home/vagrant/openfoodnetwork/
 
     - name: install node packages
-      shell: bash -lc "npm install"
+      shell: bash -lc "npm install --no-bin-links"
       args:
         chdir: /home/vagrant/openfoodnetwork/
 

